sklearn/gaussian_process/kernels.py

```diff
@@ -1821,7 +1821,7 @@ class RationalQuadratic(StationaryKernelMixin, NormalizedKernelMixin, Kernel):
     --------
     >>> from sklearn.datasets import load_iris
     >>> from sklearn.gaussian_process import GaussianProcessClassifier
-    >>> from sklearn.gaussian_process.kernels import Matern
+    >>> from sklearn.gaussian_process.kernels import RationalQuadratic
```

Comment:
I don't really get how the CI did not catch this issue.

Reply:
Everything that is imported in the module is available in the test, [Python doc reference](https://docs.python.org/3/library/doctest.html#doctest.testmod).

Opened #21186 to actually check the examples correctly in tests.